packages/ui/src/working-groups/components/Activities/WorkerRewardAmountUpdatedContent.tsx

```diff
@@ -7,10 +7,10 @@ import { WorkerRewardAmountUpdatedActivity } from '@/working-groups/types'
 export const WorkerRewardAmountUpdatedContent: ActivityContentComponent<WorkerRewardAmountUpdatedActivity> = ({
   activity,
 }) => {
-  const { newAmount } = activity
+  const { newAmount, member, openingTitle } = activity
   return (
     <>
-      Your reward amount has been updated to <TokenValue value={newAmount} />.
+      {member.handle}'s reward for {openingTitle} changed to <TokenValue value={newAmount} /> joy per block.
```

Comment:
can we user the currency constant here as well?

Reply:
[currency](https://github.com/Joystream/pioneer/blob/dev/packages/ui/src/app/constants/currency.ts) is actually part of [TokenValue](https://github.com/Joystream/pioneer/blob/dev/packages/ui/src/common/components/typography/TokenValue.tsx#L49) already, removing it here. Also merging mainnet because no need to see `tJOY` anymore. :wink: